asterics-web-vue/package.json

```diff
@@ -15,19 +15,14 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.0",
-    "@fortawesome/free-brands-svg-icons": "^5.1.0",
-    "@fortawesome/free-regular-svg-icons": "^5.1.0",
-    "@fortawesome/free-solid-svg-icons": "^5.1.0",
-    "@fortawesome/vue-fontawesome": "^0.1.1",
-    "bootstrap": "^4.1.2",
-    "bootstrap-vue": "^2.0.0-rc.11",
     "register-service-worker": "^1.0.0",
     "vue": "^2.5.16",
     "vue-router": "^3.0.1",
+    "vuetify": "^1.1.12",
     "vuex": "^3.0.1"
   },
   "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.2.0",
```

Comment:
I've forgot to remove them. Thanks!

Reply:
Sorry. I was wrong.

This dependency (fontawesome-free) still remains, but I'm a little bit confused about what is considered a development dependency and what a normal one. Do you mind explaining it?

Reply:
as I understand, the package is only needed during the build process, but not for the final result. But I can't imagine that a font would not be needed in the deployment, unless it is substituted by an svg or so.

Reply:
I had the same idea in mind. But do we need the components listed at `dependencies` or could they be moved to `devDependencies` too?

Reply:
yes, it is like @deinhofer said. `babel` is a typical `dev-dependency`, because it is only needed for building / transpiling, but not at runtime. `vue` is a typical `dependency`, because it is needed at runtime (runs in the browser). the only thing that remains where I'm not sure if it is a true `dependency` is `register-service-worker`. Do you know what it is.